src/utils/verifiable-credential/snapVC.test.ts

```diff
@@ -1,78 +1,83 @@
 import { SnapVerifiableCredential } from './snapVC';
 import { VALID_ACCOUNT_1, SNAP_SHASUM_1 } from '../test-utils';
 
+const typedDataStruct = {
+  EIP712Domain: [
+    { name: 'name', type: 'string' },
+    { name: 'version', type: 'string' },
+    { name: 'chainId', type: 'uint256' },
+  ],
+  StatusReason: [
+    {
+      name: 'type',
+      type: 'string',
+    },
+    {
+      name: 'value',
+      type: 'string[]',
+    },
+  ],
+  CredentialSubject: [
+    {
+      name: 'id',
+      type: 'string',
+    },
+    {
+      name: 'currentStatus',
+      type: 'string',
+    },
+    {
+      name: 'statusReason',
+      type: 'StatusReason',
+    },
+  ],
+  VerifiableCredential: [
+    {
+      name: '@context',
+      type: 'string[]',
+    },
+    {
+      name: 'type',
+      type: 'string[]',
+    },
+    {
+      name: 'issuer',
+      type: 'string',
+    },
+    {
+      name: 'credentialSubject',
+      type: 'CredentialSubject',
+    },
+    {
+      name: 'issuanceDate',
+      type: 'string',
+    },
+  ],
+};
+
+const typedDataDomain = {
+  name: 'VerifiableCredential',
+  version: '1',
+  chainId: 1,
+};
+
 describe('SnapVerifiableCredential', () => {
   const buildSnapVerifiableCredential = () => {
     return new SnapVerifiableCredential(1);
   };
 
-  const types = {
-    EIP712Domain: [
-      { name: 'name', type: 'string' },
-      { name: 'version', type: 'string' },
-      { name: 'chainId', type: 'uint256' },
-    ],
-    StatusReason: [
-      {
-        name: 'type',
-        type: 'string',
-      },
-      {
-        name: 'value',
-        type: 'string[]',
-      },
-    ],
-    CredentialSubject: [
-      {
-        name: 'id',
-        type: 'string',
-      },
-      {
-        name: 'currentStatus',
-        type: 'string',
-      },
-      {
-        name: 'statusReason',
-        type: 'StatusReason',
-      },
-    ],
-    StatusCredential: [
-      {
-        name: '@context',
-        type: 'string[]',
-      },
-      {
-        name: 'type',
-        type: 'string[]',
-      },
-      {
-        name: 'issuer',
-        type: 'string',
-      },
-      {
-        name: 'credentialSubject',
-        type: 'CredentialSubject',
-      },
-    ],
-  };
-
-  const domain = {
-    name: 'EIP712TrustCredential',
-    version: '1',
-    chainId: 1,
-  };
-
   describe('buildEndosedPayload', () => {
     it('builds a valid verifiable credential', () => {
       const vc = buildSnapVerifiableCredential();
       expect(
         vc.buildEndosedPayload(VALID_ACCOUNT_1, SNAP_SHASUM_1, ['reason']),
       ).toStrictEqual({
-        domain,
+        domain: typedDataDomain,
         message: {
           '@context': ['https://www.w3.org/2018/credentials/v2'],
           type: ['VerifiableCredential', 'StatusCredential'],
```

Comment:
Need to change `StatusCredential` to `ReviewCredential`
```suggestion
          type: ['VerifiableCredential', 'ReviewCredential'],
```